app/views/admin/freelancers/index.html.erb

```diff
@@ -12,7 +12,8 @@
       <h4 class="card-title">Name: <%=freelancer.name%></h4>
       <p class="card-text"><%=freelancer.description%></p>
       <p class="card-text"><%=freelancer.price%>/hr</p>
-      <%= button_to "Choose Me!", cart_index_path(freelancer_id: freelancer.id), class: "btn btn-primary"%>
+      <p class="card-text"><%=freelancer.status%>/hr</p>
```

Comment:
this shows up as "active/hr" in the view

Reply:
Great catch! fixed it and pushed it back up

Reply:
Thanks!